py-polars/polars/expr/expr.py

```diff
@@ -3702,20 +3702,15 @@ def apply(
         strategy: ApplyStrategy = "thread_local",
     ) -> Self:
         """
-        Apply a custom/user-defined function (UDF) in a GroupBy or Projection context.
+        Apply a custom/user-defined function (UDF) to each element of a Series.
```

Comment:
When I was newer to Polars, it was confusing to me when expressions were talking about operating on Series. Now I now that expressions operate on Series in the background, but maybe we can avoid potential confusion by simply referring to 'columns' here?

```suggestion
        Apply a custom/user-defined function (UDF) to each element of a column.
```